src/patternfly/components/List/examples/List.md

```diff
@@ -77,6 +77,77 @@ cssPrefix: pf-c-list
 {{/list}}
 ```
 
+### With horizontal rules
+```hbs
+{{#> list list--type="ul" list--modifier="pf-m-plain pf-m-bordered"}}
```

Comment:
```suggestion
{{#> list list--modifier="pf-m-plain pf-m-bordered"}}
```

You shouldn't need to specify `list--type`, its a `<ul>` by default - https://github.com/patternfly/patternfly/blob/bc15396391eb0414d45b51629b79b26610b14b7f/src/patternfly/components/List/list.hbs#L1